.github/workflows/macos_unit_tests.yaml

```diff
@@ -12,13 +12,16 @@ on:
 jobs:
   build:
     runs-on: macos-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9]
```

Comment:
I think some Mac OS distributions are using 3.8 now but not 3.9. I think if we only test one it would be ideal to test the one that is more-widely used.

Reply:
In that case I would only test 3.8, that's the version that ships on the latest macOS.